pkg/kritis/review/review_test.go

```diff
@@ -131,16 +132,17 @@ func TestReview(t *testing.T) {
 				PublicKeyData:        sec.PublicKey,
 			}}}, nil
 	}
-	testValidate := func(isp v1beta1.ImageSecurityPolicy, image string, client metadata.Fetcher) ([]securitypolicy.Violation, error) {
+	testValidate := func(isp v1beta1.ImageSecurityPolicy, image string, client metadata.Fetcher) ([]policy.Violation, error) {
```

Comment:
By the name here, I assumed this was implementing a test, and I didn't quite understand it. Should this be instead named mock or fake?

Reply:
sure will do